client/components/LearnerViewDirective.js

```diff
@@ -517,9 +534,29 @@ tie.directive('learnerView', [function() {
           loadQuestion(questionId, questionSet.getIntroductionParagraphs());
         };
 
+        var displayAutoSaveTextForSeconds = function(displaySeconds) {
+          $scope.displayAutoSaveText = true;
+          $scope.$apply();
```

Comment:
Interesting, now, it's working without $scope.$apply(). Not sure if it's because I used $interval instead of setInterval(). Will play around it.